WardrobeCloset/ApplicationInsights.config

```diff
@@ -0,0 +1,79 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's this kind of stuff that I don't know how to comment on :) looks very Microsoft-specific.